security/rbac/noop.go

```diff
@@ -0,0 +1,53 @@
+package rbac
+
+import "errors"
+
+var errNotEnabled = errors.New("rbac is not enabled")
```

Comment:
@speza Shouldn't a no-op RBAC return nil instead? Wouldn't this mean that normal operations which require RBAC fail rather than being ignored as the OFF switch suggests?

Reply:
@Skarlso my thoughts here are that we should return an error for any explicit RBAC operations. E.g. trying to add a role to a user, remove a role, etc - purely so those actions don't display as successful (that would be misleading). I'd rather us inform the user that RBAC is not enabled. 

For `Enforce` its slightly different, we return no error so enforcement is successful since we don't want to block endpoints when RBAC isn't on.